app/res/layout/fragment_setup_install.xml

```diff
@@ -1,96 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-                xmlns:SquareButtonWithText="http://schemas.android.com/apk/res-auto"
-                android:id="@+id/screen_first_start_main"
-                android:layout_width="fill_parent"
-                android:layout_height="fill_parent"
-                android:orientation="vertical"
-                android:background="@color/cc_core_bg"
-    >
-    <ScrollView
-        android:layout_width="fill_parent"
-        android:layout_height="wrap_content"
-        android:background="@color/cc_core_bg"
-        android:scrollbars="none">
-        <LinearLayout
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:orientation="vertical">
-
-        <!--&lt;!&ndash; TODO: extract style definition &ndash;&gt;-->
-        <include layout="@layout/grid_header_top_banner"/>
-        <LinearLayout
-            android:layout_width="match_parent"
-            android:layout_height="match_parent"
-            android:orientation="vertical"
-            android:gravity="center_horizontal"
-            >
-            <TextView
-                android:id="@+id/str_setup_message"
-                android:layout_width="fill_parent"
-                android:layout_height="wrap_content"
-                android:layout_marginLeft="@dimen/content_min_margin"
-                android:layout_marginRight="@dimen/content_min_margin"
-                android:gravity="center"
-                android:text="Welcome to CommCare!"
-                android:textSize="@dimen/text_small"
-                android:textStyle="bold"/>
-            <TextView
-                android:id="@+id/str_setup_message_2"
-                android:layout_width="fill_parent"
-                android:layout_height="wrap_content"
-                android:layout_marginLeft="@dimen/content_min_margin"
-                android:layout_marginRight="@dimen/content_min_margin"
-                android:layout_marginBottom="@dimen/content_min_margin"
-                android:gravity="center"
-                android:text="Please choose an installation method below."
```

Comment:
I removed occurrences of static text like this because we (should) always overwrite it with localized text dynamically.